Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "PackageExtensionCompat"
 uuid = "65ce6f38-6b18-4e1d-a461-8949797d7930"
 authors = ["Christopher Doris <github.com/cjdoris>"]
-version = "1.0.1"
+version = "1.1.0"
```

Comment:
1.0.2? This PR doesn't change the API.

Reply:
Whatever you say, this is your package :-). I thought that perhaps this isn't just a simple patch, and therefore at least a minor version increase would be due to indicate that, even though there is no change in the API. But if you would be more comfortable with a patch version increase, than that's whay you get.